handlers.go

```diff
@@ -3,11 +3,14 @@ package main
 import (
 	"fmt"
 	"html/template"
+	"io"
 	"io/ioutil"
 	"net/http"
 	"os"
 	"strings"
 
+	gopath "path"
```

Comment:
I think we don't need an alias here.

Reply:
The thing is, inside the upload() function, there's a variable named `path`

Reply:
I may rename the variable and remove the alias, if you agree of course.

Reply:
Oh. You are totally right. 
It's okay as it is.